### ISSUE TITLE
How can I get 3076 real and about 115000 synthetic RIRs ?

### ISSUE THREAD

**Issue by zhang-wy15** [microsoft/DNS-Challenge#56]
In the paper and readme file, it states there are 3076 real and about 115000 synthetic RIRs got from SLR26 and SLR28. Actually, SLR28 contains SLR26. And there are only 325 real and 60000 synthetic RIRs. So how can I get 3076 real and about 115000 synthetic RIRs ?

**Comment by motus** [comment]
@zhang-wy15 indeed, `SLR28/RIRS_NOISES/simulated_rirs/` is identical to `SLR26/simulated_rirs_16k/` (checked with `diff -qr`) and I don't know why we keep both. We'll probably remove `SLR26` for the next challenge. I'll get back to you with the answer to the rest of your question.

**Comment by motus** [comment]
@hadubey can you please take a look?
